packages/palette/src/elements/Join/Join.tsx

```diff
@@ -27,7 +27,7 @@ interface JoinProps {
 export const Join: SFC<JoinProps> = ({ separator, children }) => {
   const childArray = React.Children.toArray(children) as any
 
-  return childArray.reduce((acc, curr, currentIndex) => {
+  return childArray.filter(child=> child).reduce((acc, curr, currentIndex) => {
```

Comment:
Do we need an empty space before `=>`? (Or is it actually prettier recommended style?) 🤔

Reply:
oh yes, I will remove it 😅

Reply:
Sorry for the big diff, prettier messed things up

Reply:
![picard-facepalm](https://user-images.githubusercontent.com/11945712/109156828-9cc38080-7771-11eb-8a4a-596c9d2c27aa.jpg)